### PR TITLE
init

### PR DESCRIPTION
**Pull request by tpiperatgod**
Add README.md & CONTRIBUTING.md
Signed-off-by: laminar <fangtian@yunify.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+### Language
+
+- Go
+
+  [functions-framework-go](https://github.com/OpenFunction/functions-framework-go)
+
+### Runtime
+
+- Knative
+- Keda + Dapr
```
Comment (benjaminhuo) [inline README.md:15]: Keda => KEDA

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+- HTTP
+
+  The framework creates a route that associates the "/" url path and the user function. After that, it starts the HTTP service according to the PORT parameter.
+
+- CloudEvent
+
+  The CloudEvent provides a handler for handling HTTP requests while binding user function. The framework creates a route that associates the "/" url path and the CloudEvent HTTP handler. After that, it starts the HTTP service according to the PORT parameter.
+
+- Data Binding
```
Comment (benjaminhuo) [inline README.md:27]: Data Binding doesn't look like a function type, maybe we can call it something else?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
-# functions-framework
-functions-frameworks overview
+# OpenFunction Functions Framework
+OpenFunction Functions Framework is designed to provide users of FaaS with a suite of tools to convert user code into application that can be adapted to multiple runtimes.
```
Comment (benjaminhuo) [inline README.md:2]: to convert user code into application that can be adapted to multiple runtimes. =>
to convert user code into application code that supports multiple runtimes.

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tpiperatgod) [reply README.md:27]: Call it "Bindings"?

Comment (benjaminhuo) [reply README.md:27]: How about OpenFunction ?

Comment (tpiperatgod) [reply README.md:27]: Ok

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**